lib/screens/explore_screens/explore/explore_screen.dart

```diff
@@ -80,6 +86,15 @@ class ExploreScreen extends StatelessWidget {
             NavigationService.of(context).navigateTo(pageCommand.route);
           } else if (pageCommand is NavigateToBuySeeds) {
             launch('$urlBuySeeds${settingsStorage.accountName}', forceSafariVC: false);
+          } else if (pageCommand is ShowUserFlagInformation) {
+            showDialog<void>(
+              context: context,
+              builder: (_) {
+                return const FlagUserInfoDialog();
+              },
```

Comment:
I know you dont like arrow functions but it would reduce 3 lines to one :)

Reply:
haha i do like arrow functions. But we need to find a lint to catch it.

Reply:
In general I follow a rule for everything:

if it can fits in one line I'll proceed to do it :D